code/game/machinery/vending.dm

```diff
@@ -4117,11 +4117,11 @@ var/global/list/obj/item/weapon/paper/lotto_numbers/lotto_papers = list()
 
 /obj/machinery/vending/art
 	name = "\improper Le Patron des Arts"  //Mécène won't render
-	desc = "A vendor for art supplies. Pas sans mon béret !"
+	desc = "A vendor for art supplies. Pas sans mon béret!"
```

Comment:
This is correct as-is, in French a space is inserted before a two-part punctuation

Reply:
Fixed.